packages/icon/src/IconBase.ts

```diff
@@ -26,7 +26,7 @@ export class IconBase extends SpectrumElement {
         return [iconStyles];
     }
 
-    @property()
+    @property({ type: String, reflect: true })
```

Comment:
What is the benefit of reflecting here?

Reply:
removing as its not included in the issue and should be filed separately with more context